python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -368,10 +368,9 @@ def selectStars(self, exposure, sourceCat, matches=None):
         dumpData = lsstDebug.Info(__name__).dumpData                   # dump data to pickle file?
 
         detector = exposure.getDetector()
-        pixToTanXYTransform = None
+        pixToTanPix = None
         if detector is not None:
-            tanSys = detector.makeCameraSys(TAN_PIXELS)
-            pixToTanXYTransform = detector.getTransformMap().get(tanSys)
+            pixToTanPix = detector.getTransform(TAN_PIXELS)
```

Comment:
Now that I'm seeing this interface in action in real code, I'm a bit bothered by how difficult it is to see what's going on.  This case is saved by the helpful `pixToTanPix` variable name, but I think `getTransform` should be replaced by a function that really says what's going on:
 - what other coordinate system is involved ("PIXELS")?
 - are we going from the given cordinate system ("TAN_PIXELS") to the implicit one ("PIXELS"), or the reverse?

To keep the meaning of the method the same, I'd recommend calling it something like `getTransformFromPixelsTo(target)`.  It's a lot more verbose, but a lot more readable.

Of course, this is all pre-existing, but it seems like a good time to fix it.

Reply:
I agree. Depending on how widely used this is, I'll either go back to `detector.getTransformMap().getTransform(fromSys, toSys)` or add a convenience method `detector.getTransform`